apps/frontend/src/libs/components/auth-wrapper.tsx/auth-wrapper.tsx

```diff
@@ -0,0 +1,15 @@
+import { QuizForm } from "~/pages/quiz/libs/components/quiz-form/quiz-form.jsx";
+
+import { Header, Sidebar } from "../components.js";
+
+const AuthWrapper: React.FC = () => {
+	return (
+		<>
+			<Header />
+			<Sidebar />
+			<QuizForm />
+		</>
```

Comment:
Make sure to keep styling, add a screenshot to this PR

Reply:
the root page look like this

![Screenshot from 2024-08-26 20-51-52](https://github.com/user-attachments/assets/f25af7d0-a01d-44e5-a7a0-0825d8a58614)

Reply:
The layout is not correct. Sidebar should be on the left side

Reply:
i try to make the style for auth wrapper and look like this

![Screenshot from 2024-08-26 22-16-24](https://github.com/user-attachments/assets/e97a0646-33fe-4035-8e1c-4dd5f30819c5)